### ISSUE TITLE
Write documentation on how to get stacktrace of running geth

### ISSUE THREAD

**Issue by ligi** [ethereum/go-ethereum#20198]
can be helpful for issues like #20076
would be nice to have some documentation so we can point people to it

**Comment by MariusVanDerWijden** [comment]
# Creating a stack trace
A stack trace provides a very detailed look into the current state of the geth node. It helps us to debug issues easier as it contains information about what is currently done by the node. You can create a stack trace by running `debug.stacks()` in the console of your geth node. 
If you started geth without the `console` command or with a script in the background, you can use  the following command to dump the stack trace into a file. 
```
geth attach /path/to/geth.ipc --exec "debug.stacks()" > stacktrace.txt
```
You need to replace `/path/to/geth.ipc` with the path to the IPC endpoint. Geth logs the location of the IPC endpoint on startup. It is typically under `/home/user/.ethereum/geth.ipc` or `/tmp/geth.ipc.

Please don't just dump the stack trace in an issue as it makes it very hard to read. You can upload the stack trace to a gist or pastebin and put the link into the issue.

**Comment by MariusVanDerWijden** [comment]
@jmcook1186 Could you add this to the docs pls?

**Comment by s1na** [comment]
Fixed via #26106
